assets/responsibleai/tabular/components/src/create_rai_insights.py

```diff
@@ -177,6 +177,10 @@ def main(args):
             model_path=args.model_input,
         )
 
+    # unwrap the model if it's an sklearn wrapper
+    if model_estimator.__class__.__name__ == "_SklearnModelWrapper":
```

Comment:
Nit: maybe enum for the string?

Reply:
You probably mean constant?
For consistency with the preview repo I'm going to leave this unaddressed. We can add it in the next PR.